proof/drefine/Corres_D.thy

```diff
@@ -11,7 +11,7 @@
  *)
 
 theory Corres_D
-imports StateTranslationProofs_DR
+imports StateTranslationProofs_DR Lib.Corres_Method
```

Comment:
Nice to see us starting to gradually improve `drefine` as well with the `corres` method!